### PR TITLE
Add some flake8 plugins

### PR DESCRIPTION
**Pull request by wRAR**

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,8 +1,63 @@
 [flake8]
 
 max-line-length = 119
-ignore = E203, E501, E701, E704, W503
+ignore =
+    # black disagrees with flake8 about these
+    E203, E501, E701, E704, W503
 
+    # Assigning to `os.environ` doesn't clear the environment.
+    B003
```
Comment (wRAR) [inline .flake8:9]: This looks valid, I didn't check how do we use os.environ, maybe we should pass it to subprocesses explicitly as it suggests.

File: .flake8 [hunk 2]
```diff
@@ -1,8 +1,63 @@
...
+    # Do not use mutable data structures for argument defaults.
+    B006
```
Comment (wRAR) [inline .flake8:11]: I didn't check the code.

File: .flake8 [hunk 3]
```diff
@@ -1,8 +1,63 @@
...
+    # Loop control variable not used within the loop body.
+    B007
```
Comment (wRAR) [inline .flake8:13]: No opinion about this (also I didn't check the code).

File: .flake8 [hunk 4]
```diff
@@ -1,8 +1,63 @@
...
+    # Do not perform function calls in argument defaults.
+    B008
```
Comment (wRAR) [inline .flake8:15]: May be valid.

File: .flake8 [hunk 5]
```diff
@@ -1,8 +1,63 @@
...
+    # return/continue/break inside finally blocks cause exceptions to be
+    # silenced.
+    B012
+    # Star-arg unpacking after a keyword argument is strongly discouraged
+    B026
```
Comment (wRAR) [inline .flake8:20]: I checked the code but didn't think if it can/should be changed.

**Review by Gallaecio — APPROVED** [review]
I’m OK with merging as is and solving the remaining issues separately later on.

**Pull request merged**